launch/test/launch/substitutions/test_anon_name.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the AnonName substitution class."""
+
+import re
+
+from launch import LaunchContext
+from launch.substitutions import AnonName
+
+# import pytest
+
+
+def test_this_launch_file_path():
+    anon_regex = re.compile(r'.+_[0-9]+_[0-9]+')
```

Comment:
@Blast545 consider completing the pattern at each site instead of going for `.+`